### PR TITLE
first version of MCH tracking + DPL workflow for testing

### PR DESCRIPTION
**Pull request by pillot**

### CODE REVIEW

**Comment by pillot** [comment]
Hi Laurent,

I documented the code itself, but for the moment I would not have much to write in a readme. ;-) I will do that later yes.
Concerning the verbosity, it is only in (brute force) debug mode. In normal running mode it is deactivated.

File: Detectors/MUON/MCH/Tracking/src/TrackFinderOriginal.cxx [hunk 1]
```diff
@@ -0,0 +1,1346 @@
...
+{
+  /// print the track parameters if the debug level is > 1
+  if (mDebugLevel > 1) {
+    trackParam.print();
+  }
+}
+
+//_________________________________________________________________________________________________
+template <class... Args>
+void TrackFinderOriginal::print(Args... args) const
```
Comment (shahor02) [inline Detectors/MUON/MCH/Tracking/src/TrackFinderOriginal.cxx]: Why not to use FairLogger ?

**Comment by aphecetche** [comment]
@pillot during your absence the Cmake migration branch was merged (just a few hours ago, in fact), so there is now a conflict. I can help you to fix it if needed.

File: Detectors/MUON/MCH/Tracking/src/TrackFinderOriginal.cxx (hunk 1 above)
Comment (pillot) [reply Detectors/MUON/MCH/Tracking/src/TrackFinderOriginal.cxx]: Hi Ruben,

The idea was to have a first working version of the tracking, basically taken from AliRoot (in which TMatrixD were used), that I will now try to improve. Using SMatrix might help indeed, thanks a lot for the suggestion!

About the print method, I wanted a local debugging mode not to pollute (or be polluted by) general debug messages, and without having "if" statements all over the place in my code, so I found this simple solution. But if there is a more generic way to do it in O2 I will move to it for sure. Anyway this "brute force" debugging is useful for development but it does not meant to stay like this.

Cheers,
Philippe

**Comment by pillot** [comment]
I applied the changes from Laurent (thanks!) locally and it works fine. Just a couple of more checks and I push the new version.

**Comment by pillot** [comment]
Hi, the conflict is solved and all relevant tests are now green, so I suppose it can be merged.

**Comment by shahor02** [comment]
@pillot OK, will squash incremental fixes during merging

**Pull request merged**